nemo/collections/asr/parts/conformer_modules.py

```diff
@@ -141,7 +142,13 @@ def __init__(self, d_model, kernel_size):
             groups=d_model,
             bias=True,
         )
-        self.batch_norm = nn.BatchNorm1d(d_model)
+        if norm_type == "batch_norm":
+            self.batch_norm = nn.BatchNorm1d(d_model)
+        elif norm_type == "group_norm":
+            self.batch_norm = nn.GroupNorm(num_groups=1, num_channels=d_model)
```

Comment:
Same comments as above. Btw - why is it "group_norm" when in fact this is the "layer" norm settings with num_groups=1?

Reply:
dropped it. group_norm with 1 group is eq to layer_norm when both last two dimensions are passed to the layer, not just the last layer (like the ones used in other layers). group_norm here normalizes over both the time and feature.